robot/src/main/java/us/ilite/lib/drivers/SparkMaxFactory.java

```diff
@@ -0,0 +1,75 @@
+/*----------------------------------------------------------------------------*/

```

Comment:
is ILITE code really copyrighted by FIRST?

Reply:
The header is automatically added when you create a new class in VSCode, would you like me to remove it?

Reply:
Yes please

Reply:
(not a blocker to this review)